site/about.html

```
@@ -0,0 +1,225 @@
+  </div>
+</div>
+
+<h2>How It Works</h2>
+
+<div class="diagram">
+<pre style="margin:0; color: var(--muted);">
+YAML Config                    Quest Engine (TypeScript)
++-----------------------+      +-------------------------+
+| models:               |      |  .qm quest file         |
+|   - claude-sonnet-4   |      |  branching narrative     |
+|   - gpt-5.4-mini      | ---> |  state machine          |
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)